src/ol/events/condition.js

```diff
@@ -245,5 +245,5 @@ export const mouseOnly = function(mapBrowserEvent) {
  */
 export const primaryAction = function(mapBrowserEvent) {
   const pointerEvent = mapBrowserEvent.pointerEvent;
-  return pointerEvent.isPrimary && pointerEvent.button === 0;
+  return pointerEvent !== undefined && pointerEvent.isPrimary && pointerEvent.button === 0;
```

Comment:
I wonder if we'll want `instanceof MapBrowserPointerEvent` here instead for type checking.  Also curious about the performance compared to this duck typing.